tests/client/amo/actions/test_landing.js

```diff
@@ -0,0 +1,46 @@
+import * as actions from 'amo/actions/landing';
+
+
+describe('LANDING_GET', () => {
+  const action = actions.getLanding({ addonType: 'theme' });
+
+  it('sets the type', () => {
+    assert.equal(action.type, 'LANDING_GET');
+  });
+
+  it('sets the filters', () => {
+    assert.deepEqual(action.payload, { addonType: 'theme' });
+  });
+});
+
+describe('LANDING_LOADED', () => {
+  const entities = sinon.stub();
+  const response = {
+    featured: sinon.stub(),
+    highlyRated: sinon.stub(),
+    popular: sinon.stub(),
+  };
+  const action = actions.loadLanding({
+    addonType: 'theme', entities, ...response });
+
+  it('sets the type', () => {
+    assert.equal(action.type, 'LANDING_LOADED');
+  });
+
+  it('sets the payload', () => {
+    assert.deepEqual(action.payload, {
+      addonType: 'theme', entities, ...response });
```

Comment:
again, `entities` can be removed.

This is an example of where I think you'll get better tests by following a pattern like: set data using an action -> make assertions about how the *reducer stored the data*. Instead, what you're doing is making assertions about the action payload which isn't so helpful.